process/smartContract/scDataGetter.go

```diff
@@ -0,0 +1,103 @@
+package smartContract
+
+import (
+	"fmt"
+	"math"
+	"math/big"
+	"sync"
+
+	"github.com/ElrondNetwork/elrond-go/process"
+	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
+	"github.com/pkg/errors"
+)
+
+var maxGasValue = big.NewInt(math.MaxInt64)
+
+// scDataGetter can execute Get functions over SC to fetch stored values
+type scDataGetter struct {
```

Comment:
rename type. it is returning vmOutput.ReturnData. -> scReturnDataHandler

Reply:
I think that the current naming explains better what it does. Waiting for the second reviewer's opinion.